docs/recipes/recipe-marketing-attribution/index.md

```diff
@@ -225,25 +193,132 @@ FROM derived.marketing_attribution
+
+The custom `conversion` event used in this recipe is very flexible so that you can instrument it across as many or as few conversions as possible. Specifically, it has the following fields:
+
+<table><tbody><tr><td><strong>Field</strong></td><td><strong>Description</strong></td><td><strong>Type</strong></td><td><strong>Validation</strong></td><td><strong>Required?</strong></td></tr><tr><td><code>name</code></td><td>The name of the conversion</td><td>string</td><td><code>maxLength: 255</code></td><td>❌</td></tr><tr><td><code>value</code></td><td>The value assigned to the conversion, such as the revenue associated with it</td><td>integer</td><td><code>minimum: 0</code>,<br/><code>maximum: 1000000</code><br/></td><td>❌</td></tr></tbody></table>
+
+#### Implementing the event
+
+Trigger the conversion events wherever you have conversions on your site. Some examples might be:
+
+- Newsletter sign up
+- Cart checkout
+- Item download
+
+#### In the JavaScript Tracker
+
+```javascript
+window.snowplow('trackSelfDescribingEvent', {
+   "event": {
+      "schema": "iglu:io.snowplow.foundation/conversion/jsonschema/1-0-0",
+      "data": {
+         "name": "email-signup",
+         "value": 10
+      }
+   }
+});
+```
+
+#### Via Google Tag Manager
+
+If you are using Google Tag Manager, you can add the variables like so:
+
+```javascript
+window.snowplow('trackSelfDescribingEvent', {
+   "event": {
+      "schema": "iglu:io.snowplow.foundation/conversion/jsonschema/1-0-0",
+      "data": {
+         "name": "{{example_conversion_variable}}",
+         "value": {{example_value_variable}}
+      }
+   }
+});
+```
+</details>
+
+## 3. Connect to your data warehouse
+
+Now that you've seen the principles in action, you've actually done the hardest part.
+
+Now you just need to connect to your warehouse and starting building up an asset of customer data that you can get better and better attribution insights out of over time!
+
+To connect to your warehouse / lake, you need to go into the Snowplow UI, click 'Upgrade Now' and follow the setup wizard for BDP Cloud (you will need a credit card for this), this is a production-ready platform that can reliably load live events to your warehouse / lake.
+
+![](images/destination.png)
+
+When your connection is confirmed, deploy our prebuilt dbt packages following our quickstart guides:
+
+1. Snowplow [web package](https://docs.snowplow.io/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/web/)
+2. Snowplow [attribution package](https://docs.snowplow.io/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/fractribution/)
```

Comment:
How do Try customers deploy models, can they do it via BDP or would they have to run and manage these locally?